app/controllers/posts_controller.rb

```diff
@@ -24,10 +24,10 @@ def show
     mark_corresponding_notifications_read if user_signed_in?
 
     respond_to do |format|
-      format.html{ gon.post = PostPresenter.new(@post, current_user); render 'posts/show', layout: 'with_header_with_footer' }
-      format.xml{ render :xml => @post.to_diaspora_xml }
-      format.mobile{render 'posts/show' }
-      format.json{ render :json => PostPresenter.new(@post, current_user) }
+      format.html{ gon.post = PostPresenter.new(@post, current_user); render "posts/show" }
```

Comment:
Do not use semicolons to terminate expressions.<br>Space missing to the left of {.